Portofolio Add a contact form and finish the mobile version PR

diarisdiakite opened the pull request:
In this I Added a contact form in pair programming with my teammates:
- Created the form and added the styling
- Fixed styling issue with the project cards 
- Fixed linters Css issues

didierganthier reviewed (commented):
Hi Team,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.

<img src="https://media.giphy.com/media/HOTfexLzbar5u/giphy.gif" />

## Invalid Code Review Request 🚫

- You have submitted a project with a README badly formatted, please respect the same format of the [template](https://github.com/microverseinc/readme-template/blob/master/README.md), while keeping the `Table of contents` and the MIT License link. 

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

braincee commented on index.html:
```diff
@@ -268,14 +271,67 @@ <h3>
+      </h2>
+    </div>
+
+    <form action="https://formspree.io/f/mqkoykyv" method="POST" class="contact-form-container">
+
+      <label>
+        <input type="text" name="full-name" placeholder="Full name">
+      </label>
+        
+      <label>
+        <input type="email" name="email" placeholder="Email address">
+      </label>
+        
+        
+      <textarea name="" id="" rows="10" placeholder="Hey, I would like to connect to..."></textarea>
+        
+        
+      <button type="submit"> Get in touch </button>
+
+    </form>
```
- [ ] Please I highly recommend implementing HTML validations to the form field per project requirements. 😃 
   - Name input (can be first name and last name field depending 
      on the template used) has a `maxlength="30"` attribute.
    - Message input has `maxlength="500"` attribute.
    - All input fields have the `required` attribute to activate the 
       validation on submission.
    - Please this is an added screenshot of the specific project 
      requirements. 😃 

![Screenshot (119)](https://user-images.githubusercontent.com/78104544/228726499-3ccf44f8-0af8-4627-a714-22529305ddc4.png)

braincee commented on css/styles.css:
```diff
@@ -53,6 +54,15 @@ So I grouped all my buttons here
   width: max-content;
 }
 
+button {
+  background-color: var(--my-project-boutton-color);
+  color: white;
+  padding: 3% 3%;
+  margin: 3% auto 10% auto;
+  width: max-content;
+  border-radius: 4%;
+}
```
- [ ] Please I kindly suggest adding button effects (enable, hover, pressed, etc.) per project requirements. 😄 

![Screenshot (121)](https://user-images.githubusercontent.com/78104544/228727560-715337cb-1834-4afa-86f9-b404a25a4372.png)

Please this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) is provided for you to cross-check the project requirements. 😅

diarisdiakite replied:
Done @braincee Thanks

braincee commented on README.md:
```diff
@@ -1,62 +1,72 @@
 (back to top)
 
-## 👥 Authors
+# 👥 Authors
 
 👤 Didy
 
```
- [ ] [OPTIOANL] I kindly suggest adding pair-programming partners including social media handles to the authors' section. 😃

diarisdiakite replied:
Done @braincee.
However, I had already added the name of one collaborator in the Acknowledgement. Do they really need to be Authors to in my portfolio repo?

braincee reviewed (changes requested):
Hello team @diarisdiakite, @romans-adi, and @BenMKT  ✋, I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close](https://user-images.githubusercontent.com/78104544/225492039-ed22bba9-481a-489f-ae04-74623970af20.gif)

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Great implementation of pair programming ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

diarisdiakite commented:
In this milestone, I:
- fixed button issues making them expand to the entire with of container,
- Added the button interactions for hover, enabled and active,
- Added the required and maxlength properties for my input.s and textarea fields (according to the template requirements)
- Added my coding partners as contributors in the author section (but the question is still pending)

huseyinbiyik commented on index.html:
```diff
@@ -268,14 +271,65 @@ <h3>
+        
+      <textarea name="" id="" rows="10" maxlength="500" placeholder="Hey, I would like to connect to..." required></textarea>
+        
+      <div class="form-button" type="submit"> <a href="https://formspree.io/f/mqkoykyv">Get in touch</a> </div>
```
Unfortunately, your form is not submitting and giving a POST error, also validations are not working. You need to set up a [formspree](https://formspree.io/html/) account to get an email from this contact form. And please don't use the `<a>` element instead of a `<button>`. It was correct before the previous review, now you turned it to the `<a>` link but it is incorrect. When you change it back to a submit button and get a form spree account, it is going to be fine. Please remember to test your validation and submission before asking for a review 👍

diarisdiakite replied:
Well received @huseyinbiyik. I made the changes. Thank you for the comment, the form is now submitting to my formspree account.

huseyinbiyik reviewed (changes requested):
Hi team,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![tenor](https://user-images.githubusercontent.com/25887253/205415161-00e79d1b-af68-4f10-9b2c-265f460dc5da.gif)

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

diarisdiakite commented:
In this milestone, I:
- Got back to the button type submit to submit my form
- resolved the submission issue to formspree

brainconnect93 reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @diarisdiakite, 👋 👋  

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 


<img src="https://user-images.githubusercontent.com/85113567/184604589-430cf4bc-1d2e-4603-976a-7a48a899ea35.gif" width="600px" height="300px">

## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

- [ ] [OPTIONAL] Kindly add the `Table of content` to your readme file in order to make it more professional and descriptive and also for a better transitioning around your readme file

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
